app/views/documents/Urlaubsantrag.html.erb

```diff
@@ -42,14 +42,15 @@
  </tr>
 
 <tr style='height:17pt'>
- 	<td width=30><%= holiday.previous_year %></td>
- 	<td width=30><%= holiday.actual_year %></td>
- 	<td width=30><%= holiday.obtained_days %></td>
- 	<td width=30><%= holiday.remainig_days %></td>
- 	<td width=110><b><%= holiday.start_date %></b></td>
- 	<td width=110><b><%= holiday.end_date %></b></td>
- 	<td width=30><b><%= holiday.duration %></b></td>
- 	<td width=30><%= holiday.new_remainig_days %></td> <!-- holiday.remainig_days=holiday.remainig_days-holiday.duration%> -->
+ 	<td width=30><%= User.find(holiday.user_id).remaining_leave_last_year %></td>
+ 	<td width=30><%= User.find(holiday.user_id).remaining_leave - User.find(holiday.user_id).remaining_leave_last_year %></td>
+ 	<td width=30><%#= holiday.obtained_days %></td>
+ 	<td width=30><%#= holiday.remainig_days %></td>
+ 	<td width=110><b><%= holiday.start.strftime("%d.%m.%Y") %></b></td>
+ 	<td width=110><b><%= holiday.end.strftime("%d.%m.%Y") %></b></td>
```

Comment:
Beim Datepicker gibt es einen harten Bug: wenn schon ein Datum drin steht und man wieder in das Feld klickt wird das datum falsch rum gelesen und der datepicker springt auf irgendwann 19xx - liegt wahrscheinlich daran dass er das Datumsfeld in falscher Reihenfolge ausliest.